phpstan.neon

```diff
@@ -31,14 +27,16 @@ parameters:
 	level: 7
 	reportMaybesInPropertyPhpDocTypes: false
 	checkMissingIterableValueType: false
+	checkDynamicProperties: false
```

Comment:
Why that? Do you have a specific example where PHPStan reports an error if we do not disable the check and which cannot be solved otherwise? I deem dynamic properties as one of the top causes for bugs. If we have dynamic properties (e.g. like in the models) we should really annotate them with `@property` to ensure that they are recognized and correctly spelled everywhere.

Reply:
With :
```
	# checkDynamicProperties: false
	# featureToggles:
	# 	bleedingEdge: true
```
Results:
```sh
▶ make phpstan                      
vendor/bin/phpstan analyze
Note: Using configuration file /home/biv/Documents/Lychee/phpstan.neon.
 434/434 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%
 [OK] No errors                                                                                                         
```

------------

With:
```
	# checkDynamicProperties: false
	featureToggles:
		bleedingEdge: true
```
Results:
```
▶ make phpstan
vendor/bin/phpstan analyze
Note: Using configuration file /home/biv/Documents/Lychee/phpstan.neon.
 434/434 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%
 ------ ----------------------------------------------------------------------------------------------------------------------- 
  Line   app/Actions/AlbumAuthorisationProvider.php                                                                             
 ------ ----------------------------------------------------------------------------------------------------------------------- 
  589    Access to an undefined property App\Models\Extensions\AlbumBuilder|App\Models\Extensions\FixedQueryBuilder::$columns.  
 ------ ----------------------------------------------------------------------------------------------------------------------- 
 ------ ------------------------------------------------------------------------------------ 
  Line   app/Actions/PhotoAuthorisationProvider.php                                          
 ------ ------------------------------------------------------------------------------------ 
  348    Access to an undefined property App\Models\Extensions\FixedQueryBuilder::$columns.  
 ------ ------------------------------------------------------------------------------------ 
 [ERROR] Found 2 errors                                                                                                 
make: *** [makefile:107: phpstan] Error 1
```

Bleeding edge is necessary for the enforcement of `===` and `!==`.